### DISCUSSION TITLE
Error installing any starter kit?

### DISCUSSION THREAD

**Discussion by riordanmb** [statamic/cms#8196] [Q&A]
Unable to instrall any starter kit on existing site

 php please starter-kit:install statamic/starter-kit-cool-writings 

I get this error: 

Composer could not authenticate with GitHub!
Please generate a personal access token at: https://github.com/settings/tokens/new
Then save your token for future use by running the following command:
composer config --global --auth github-oauth.github.com [your-token-here]
Error installing starter kit [statamic/starter-kit-cool-writings]. I am logged into github and have authenticated with a token and it still doesn't work. Though not sure why I need to do this?

**Comment by duncanmcclean** [comment]
Do you get any errors when you run the `composer config` command?

**Comment by riordanmb** [comment]
No I don’t.
